apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -249,10 +266,14 @@ export const CssValueInput = ({
     },
   });
 
-  const [inputRef, isInputActive] = useScrub({
+  const shouldHandleEvent = useCallback((node) => {
+    return !suffixRef.current?.contains?.(node);
```

Comment:
hard to read, don't do the !thing please

```suggestion
    return suffixRef.current?.contains?.(node) === false;
```